index.ts

```diff
@@ -65,7 +65,8 @@ export class ConfigSDK {
   constructor(childApi: ChildAPI, eventBus: EventBus) {
     this.parent = childApi;
     this.eventBus = eventBus;
-    this.config = {};
+    // @ts-ignore
+    this.config = childApi.model.config || {};
```

Comment:
The DefinitelyTyped types for postmate don't include this `.model` prop. I can't actually tell from Postmate's documentation if this is intended as a public API, so I'm not sure if I should update the type definitions. I looked into doing it some other way with an "init" event, but it was way too complicated when this just works.